EventsExpress.Core/DTOs/UserMoreInfoDTO.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
There are two files appear to this PR: UserMoreInfoDTO.cs and UserMoreInfoDto.cs
be aware that git for windows case insensitive for path

Reply:
Seems like a git issue as there is no UserMoreInfoDTO.cs in the file system. I'm looking into this